### ISSUE TITLE
Feature Request: Make the scrollbar Moveable, Followable & click/pass through able 

### ISSUE THREAD

**Issue by Droyk** [Christoph142/modern-scroll#191]
As I mentioned in #189 it's hard to follow the scrollbar if the reading content is way far on each end. & scroll bar is stuck on only one side. 
Feature suggested in #189 would work for the above annoyance but if the reading is in the [middle](http://kicherer.org/joomla/index.php/de/blog/42-avcut-frame-accurate-video-cutting-with-only-small-quality-lossso)
as you can see the main reading content is in the middle right but I've configured my scroll bar to remain on the left. scrollbar that far feels a distraction constantly moving my eyes from the content and then going back to where I left is annoying & it's not just once in a while. I stumble upon these 50 -60% of my browsing experience! by making the scroll bar follow the cursor we eliminate this problem entirely!

Make the scrollbar follow the mouse and then if the user chooses user can place the scrollbar via pressing a hotkey wherever on the page user wants to drop the scrollbar. this feature also warrants the ability to make it click-through so it doesn't get b/w the user work & a hotkey to quickly disable the effect. (better to make a quick hold and release key or a simple enable/disable key better to provide both because they are equally gonna be in need.) so whenever the users want to use the scroll bar doesn't matter to scroll or move the scrollbar on the page they can do so efficiently!  

This feature needs to provide various user choices to make the user experience better

- Followble & click-through able from the start user can place wherever they wish to. or let the scroll follow the cursor indefinitely.

- Stick to the sides & only follow if the user activates the function via the hotkey. Making it quickly follows the cursor now user can drag & drop it anywhere on the page or just let it follow the cursor however they want to use it they can use it!

- A hotkey to make move the scrollbar instantly on either side of the page.

**Comment by Droyk** [comment]
@Christoph142 Here's another example: https://www.ghacks.net/2021/12/17/pale-moon-project-announces-change-of-direction/#comments
try to immerse yourself in the comments for a bit and then try to see the scroll bar to see how much you've read! I find it's really hard to find the scroll bar and then go back to where I was

**Comment by Christoph142** [comment]
It's an interesting idea. It just needs a bit of spare time...
